### ISSUE TITLE
Wrong alignment results in usearch_global

### ISSUE THREAD

**Issue by davidealbanese** [torognes/vsearch#153]
--fastapairs and  --userout (--userfields options: qrow, trow) options in usearch_global seem to return wrong alignments. --alnout seems to work fine.

**Comment by torognes** [comment]
Thank you for reporting this problem. Could you please give some more details about the problem? A small example would be perfect. Otherwise it is difficult to reproduce the problem and fix it.

**Comment by davidealbanese** [comment]
Dear @torognes, in attachment a simple example. The command line I used is (vsearch 1.9.5 osx):
```
vsearch --usearch_global query.fasta --db db.fasta --id 0.80 --alnout aln_alnout.txt \\
    --fastapairs aln_fastapairs.fasta --userout aln_userout.txt \\
    --userfields=query+target+id+qrow+trow
```
As you can see, the alignments in `aln_fastapairs.fasta` and `aln_userout.txt ` are incorrect.

Thank you,
Davide

[example.zip](https://github.com/torognes/vsearch/files/82538/example.zip)

**Comment by torognes** [comment]
Thank you, @davidealbanese! I'll look into it right away.

**Comment by torognes** [comment]
This bug should be fixed in the new release of vsearch version 1.9.6. Thanks again for reporting the bug!

**Comment by davidealbanese** [comment]
Thank you @torognes!
